### PR TITLE
Combine information about links

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- Combine information about links

**Related issue(s)**
#362

### CODE REVIEW

File: guidelines/content-guidelines/05-links.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
 ---
-title: Links in docs
+title: Links
 ---
 
-These are the guidelines for making cross-references between the documents in the [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) folder.
+Linking is a great tool to use to incorporate a lot of content into your document with fewer words. That being said, overuse of linking can cause "link rot" when links break, and if a page has more links than content, it is not very pleasing to read.  Make sure to use links correctly by adhering to these best practices:
```
Comment (kazydek) [inline guidelines/content-guidelines/05-links.md]:
```suggestion
Linking is a great tool to use to incorporate a lot of content into your document with fewer words. That being said, overuse of linking can cause "link rot" when links break, and if a page has more links than content, it is not very pleasing to read. Make sure to use links correctly by adhering to these best practices:
```

File: guidelines/content-guidelines/04-formatting.md [hunk 2]
```diff
@@ -126,6 +126,7 @@ Ideally, headings fit into one line in the generated output. Be concise, but als
 * Use action verbs and present tense verbs in headings when possible, especially in tutorials. For example, **Add a document type**.
 * While gerunds are acceptable in body-level content, DO NOT use gerunds in headings. Use **Create a storefront** instead of **Creating a storefront**.
 * Avoid stacked headings, which are headings without body-level content in between. For example, DO NOT use a Heading 2 (H2) to introduce one or more Heading 3s. Instead, add a paragraph after the H2 that describes the main idea of the content in the headings that follow.
+<<<<<<< HEAD
```
Comment (kazydek) [inline guidelines/content-guidelines/04-formatting.md]: Hmm sth went wrong here - looks like you were trying to resolve a conflict but the content got mixed up - could you take a look at the file preview and remove the unwanted elements? 

image

Comment (bszwarc) [reply guidelines/content-guidelines/04-formatting.md]: 😠 How did this get into the doc :D We solved the conflict today and it looked good :D

Comment (bszwarc) [reply guidelines/content-guidelines/04-formatting.md]: Looks good now :D

Comment (alexandra-simeonova) [reply guidelines/content-guidelines/04-formatting.md]: Thank you both 😄

**Pull request merged**